### PR TITLE
Add recipe for sebtm/inwx-api-bundle

### PR DESCRIPTION
**Pull request by SebTM**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: sebtm/inwx-api-bundle/0.1/manifest.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "bundles": {
+        "SebTM\\INWX\\InwxApiBundle": [
```
Comment (Nyholm) [inline sebtm/inwx-api-bundle/0.1/manifest.json]: Could you write this in one line?

**Comment by SebTM** [comment]
Hey @Nyholm,

I directly access the values by "env"-function. Is there an example or documentation how this is done in best practice?

Thanks,
SebTM

**Comment by Nyholm** [comment]
I think it is up to you and your bundle to decide. If you directly access the environment variables you need to require the DotEnv component. Your way of doing it is not at all wrong. But that is not how most bundles do it. 

Im find with approving this recipe if you like to. If you want to add a config file that is also very fine.

**Comment by SebTM** [comment]
@Nyholm Would be ok if you merge it I will reconsider this for one of the next releases 👍

**Pull request merged**